<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>711</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>11</y>
       <width>591</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Set File Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Explore</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Point File Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Explore</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>LandXML File Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Explore</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create LandXML</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotBrowseSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>778</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotBrowseXML()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotBrowsePoint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotCreateLandXML()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotSetFilePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotPointFilePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_5</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotXMLFilePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSetFilePath()</slot>
  <slot>slotXMLFilePath()</slot>
  <slot>slotPointFilePath()</slot>
  <slot>slotCreateLandXML()</slot>
  <slot>slotBrowsePoint()</slot>
  <slot>slotBrowseSet()</slot>
  <slot>slotBrowseXML()</slot>
 </slots>
</ui>
